index.html

```diff
@@ -2,23 +2,71 @@
 <html lang="en">
 	<head>
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
+    	<meta name="Relaxr" content="Your description goes here">
     	<meta name="keywords" content="one,two,three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
+		
 
 		<!-- CSS links -->
+		
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400,300,600,800' rel='stylesheet' type='text/css'>
 
```

Comment:
Don't forget to add either your CSS normalize.css or reset.css  before your style.css file